src/main/java/Bobby.java

```diff
@@ -12,13 +13,13 @@ public class Bobby {
     /**
      * Greeting message displayed when the bot starts
      */
-    private static String greeting = "Hello! I'm Bobby\n"
+    private static final String GREETING = "Hello! I'm Bobby\n"
```

Comment:
Should the name of the GREETING String be changed to START to be similar to the EXIT String in terms of naming due to their similar functions?